unordered-containers.cabal

```diff
@@ -39,6 +39,7 @@ library
   build-depends:
     base >= 4 && < 5,
     deepseq >= 1.1,
+    ghc-prim >= 0.2,
```

Comment:
Not for all `base`, `GHC.Generics` is in only since `base-4.6`. Also `Generics` are added in `GHC-7.2`, yet `unordered-containers` seems to compile down to `7.0`.

I'm not sure whether @tibbe wants to keep the compatibility.